test/HubPool.Admin.ts

```diff
@@ -135,4 +135,39 @@ describe("HubPool Admin functions", function () {
   it("Only owner can set liveness", async function () {
     await expect(hubPool.connect(other).setLiveness(1000000)).to.be.reverted;
   });
+  it("Only owner can pause", async function () {
+    await expect(hubPool.connect(other).setPaused(true)).to.be.reverted;
+    await expect(hubPool.connect(owner).setPaused(true)).to.emit(hubPool, "Paused").withArgs(true);
+  });
+  it("Cannot propose while paused", async function () {
```

Comment:
For completeness, you should add a test that you can't execute a proposal while paused